src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const isDanger = person.sex === 'f';
+
+  const { mother, father } = person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({ 'has-background-warning': person.slug === slug })}
+    >
+      <td>
+        <Link
+          to={person.slug}
+          className={cn({
+            'has-text-danger': isDanger,
+          })}
+        >
+          {person.name}
+        </Link>
```

Comment:
<img width="679" alt="image" src="https://github.com/mate-academy/react_people-table-basics/assets/86789161/bd0fb35b-5f6f-4f23-9095-ef5242de4dec">

Create separate component for this and reuse it here and below